system/mp/mpbase.cpp

```diff
@@ -556,7 +556,8 @@ protected: friend class CNodeIterator;
     static IGroup *fromText(const char *s,unsigned defport) 
     {
         SocketEndpointArray epa;
-        epa.fromText(s,defport);
+        if (s)
```

Comment:
Not really necessary now that your other change means SocketEndpointArray::fromText handles NULL.